api/routes/api.php

```diff
@@ -14,6 +15,15 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+Route::group(['middleware' => 'auth:sanctum', 'prefix' => '/v1'], function () {
+    Route::get('/', function () {
+        return response()->json(["message"=>"You are at the version 1 of this api"]);
+    });
+
+    Route::group(['prefix' => 'user'], function () {
+        Route::get('/', [UserController::class, 'index']);
+        Route::get('/{user_id}', [UserController::class, 'show']);
```

Comment:
```suggestion
        Route::get('/{user}', [UserController::class, 'show']);
```

Reply:
Isn't it more readable if I only want to use the id of the user?

Reply:
anyway, I'm gonna use user, thanks!